src/okapi_em/_widget.py

```diff
@@ -230,15 +231,15 @@ def ui_do_tab_ResMeas(self):
             vBox4.addLayout(qgrid3_layout)
             qgrid3_layout.addWidget(QLabel("tile size (px):"),0,0)
             self.spbQuolTileSize_px = QSpinBox()
-            self.spbQuolTileSize_px.setMinimum(8)
+            self.spbQuolTileSize_px.setMinimum(128)
```

Comment:
why minimum of 128? Can you not run with 64 or 32?

Reply:
No, Miplib doesn't allow anything below 128